docs/api/IoT.js-API-Module.md

```diff
@@ -2,48 +2,40 @@
 
 The following shows module APIs available for each platform.
 
-|  | Linux<br/>(Ubuntu) | Raspbian<br/>(Raspberry Pi) | Nuttx<br/>(STM32F4-Discovery) |
+|  | Linux<br/>(Ubuntu) | Raspbian<br/>(Raspberry Pi) | NuttX<br/>(STM32F4-Discovery) |
 | :---: | :---: | :---: | :---: |
 | require | O | O | O |
 
-### Contents
-
-- [Module](#module)
-    - [Module Functions](#module-functions)
-        - [`require(id)`](#requireid)
-    - [Module Loading by `require` function](#module-loading-by-require-function)
-
 # Module
-`module` implements `require` function.
+This module implements `require` function.
 Since `require` function is implicitly passed to user code, user do not need to import it explicitly.
```

Comment:
I would rephrase this to:

The `require` function is always available there is no need to import `module` explicitly.